region/maridia/inner-green/Pants Room.json

```diff
@@ -629,6 +629,30 @@
         "Time another jump with the water level or walljump to fully escape the water."
       ]
     },
+    {
+      "link": [5, 2],
+      "name": "Pants Room Bomb Jump Water Escape",
+      "notable": true,
+      "requires": [
+        "HiJump",
+        "h_canMaxHeightSpringBallJump",
+        "canSpringFling",
+        "canBombJumpWaterEscape",
+        "h_canJumpIntoIBJ"
```

Comment:
Sorta want to add canDoubleBombJump since the water catches up to me half the time.
Bombing into a springballjump is halfway to being a new tech.

Reply:
It's a really weird mechanic that I thought wasn't possible. It could be useable in walljumpless hell runs, but seems a bit rough. It's also possible to kill them with a PB, but it's kinda tight.

Reply:
Yeah it looks like you can keep getting Spring ball jumps indefinitely in this way to speed up a long IBJ (at least in terms of in-game time, e.g. for heat damage)?